pkg/util/precision/converter.go

```diff
@@ -71,6 +79,9 @@ func (c *Fixed8Converter) SetBalancePrecision(precision uint32) {
 		exp = -exp
 	}
 
+	c.m.Lock()
+	defer c.m.Unlock()
```

Comment:
Frankly, I'd just internalize this `SetBalancePrecision` and leave it all to `NewConverter`.

Reply:
dont you like how every blockchain config is updated [the same way](https://github.com/notimetoname/neofs-node/blob/92c391dda95e151bc96ca55d9242b20c8406d6f4/pkg/innerring/innerring.go#L1101-L1104) (i even have not created any func for that, `initConfigFromBlockchain` already exists!)?

Reply:
Epoch-related things _can_ change without reconnecting, balance precision can't.

Reply:
> balance precision can't

oh, really? it simplifies things a little.